app/models/training_slide.rb

```diff
@@ -62,7 +62,7 @@ def self.inflate(all_content, slug, wiki_page = nil)
     slide.save
     slide
   rescue StandardError => e
-    puts "There's a problem with file '#{slug}'"
```

Comment:
Let's not change this; it's easier to notice when this particular problem happens than if it's done with the logger, I think.

Reply:
Okay, I've undone these changes.